src/dev/ci_setup/setup.sh

```diff
@@ -13,6 +13,10 @@ echo " -- TEST_ES_SNAPSHOT_VERSION='$TEST_ES_SNAPSHOT_VERSION'"
 ###
 ### install dependencies
 ###
+
+# cleanup node_modules from packer cache
+find . -type d -name node_modules -not -path '*__fixtures__*' -prune -print0 | xargs -0I % rm -rf "%"
```

Comment:
Same here?

Reply:
This needs to exist because it's in the actual packer cache, not the bootstrap cache.

Reply:
Hmm, have you verified this? I'm pretty sure the packer cache script executes in a different directory from the Jenkins workspace, so the remnants of the checkout/bootstrap work done by the packer cache will not be there